<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>780</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>LookBoard</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>780</height>
    </rect>
   </property>
   <widget class="QWidget" name="mainpage">
    <widget class="QWidget" name="topBg" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>680</width>
       <height>300</height>
      </rect>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>40</y>
        <width>100</width>
        <height>2</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0.4)
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="topLabel">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>25</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 24pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>看板</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>15</width>
        <height>2</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0, 0.4)
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="renew">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>34</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QScrollArea" name="topScrollArea">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>580</width>
        <height>200</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>580</width>
        <height>170</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="topScrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>578</width>
         <height>198</height>
        </rect>
       </property>
      </widget>
     </widget>
     <zorder>topLabel</zorder>
     <zorder>label_4</zorder>
     <zorder>label</zorder>
     <zorder>renew</zorder>
     <zorder>topScrollArea</zorder>
    </widget>
    <widget class="QWidget" name="btmBg" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>540</y>
       <width>681</width>
       <height>241</height>
      </rect>
     </property>
     <widget class="QLabel" name="btmLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>喜欢的文章</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>581</width>
        <height>2</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0.8)
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QScrollArea" name="btmScrollArea">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>580</width>
        <height>170</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>578</width>
         <height>168</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="midBg" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>680</width>
       <height>240</height>
      </rect>
     </property>
     <widget class="QLabel" name="midLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>最近阅读</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>581</width>
        <height>2</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0.8)
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QScrollArea" name="midScrollArea">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>580</width>
        <height>170</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>578</width>
         <height>168</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="articlepage">
    <widget class="QWidget" name="topWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>680</width>
       <height>61</height>
      </rect>
     </property>
     <widget class="QPushButton" name="backButton_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="likeButton_5">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/new/xin.png</normaloff>:/new/xin.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="btmWidget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>681</width>
       <height>721</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="articlename">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>431</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>浪漫雅圆</family>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="contentView">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>471</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'幼圆'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="articleImg">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>350</width>
       <height>233</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
